Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
I was trying to guess your preferred style from https://github.com/Homebrew/homebrew-core/pull/4635#discussion_r78106419 😓 Although the current version does look quite hedious.